data/multiplayer/scenarios/4p_King_of_the_Hill.cfg

```diff
@@ -54,9 +48,167 @@
         side=4
         team_name=north-west
         user_team_name= _ "teamname^Northwest"
-        canrecruit=yes
         controller=human
         gold=150
-        fog=yes
     [/side]
+
+    [options]
+        [slider]
+            id="bonus_income"
+            default=15
+            min=5
+            max=40
+            name= _ "Hill Bonus Income"
+            description= _ "Gold Per Turn To Leader On Hill Keep"
```

Comment:
```suggestion
            description= _ "The amount of extra gold per turn granted to the leader who occupies the hill keep."
```

Reply:
Okay so tool tips should just use regular grammar (including a period), I will try to keep that in mind in the future.